rakelib/lint.rake

```diff
@@ -0,0 +1,22 @@
+namespace :lint do
+  desc 'Install swiftlint'
+  task :install do |task|
+    swiftlint = `which swiftlint`
```

Comment:
I'd find it both more logical and less dangerous to do both the `which` and the `$?.success?` tests in one line.
Because:

* the purpose of `which swiftlint` isn't really to know the path where `swiftlint` is installed but rather if it's installed at all (we could use `swiftlint --version` for that test instead)
* If we happen to modify this script later and add another command execution between this `which swiftlint` and the test wether the last shell command was and error or success, then that test (`$?.success?`) would check the wrong command

I suggest we write it this way instead:

```sh
swiftlint_installed = `swiftlint --version` && $?.success?
if !swiftlint_installed
  …
```

(I'm open to options if you know a better way of testing from ruby if a command line tool exists an is installe btw)